scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/helpers/TaskResultCreator.java

```diff
@@ -110,12 +105,10 @@ public TaskResultImpl getEmptyTaskResult(InternalTask task, Throwable exception,
         Map<String, byte[]> variables = new HashMap<>();
 
         if (job.getType() == JobType.TASKSFLOW && eligibleTaskDescriptor != null) {
-            try{
-                TaskResultImpl taskResult = (TaskResultImpl) dbManager.loadTasksResults(job.getId(),
-                        Collections.singletonList(task.getId())).get(task.getId());
+            TaskResultImpl taskResult = (TaskResultImpl) dbManager.loadLastTaskResult(task.getId());
```

Comment:
The logic is clear to check the taskResult. But it still can occur the DatabaseManagerException exception, as it is thrown by loadTasksResults() method. It is needed to catch the exception.